Functions/xASL_num2str.m

```diff
@@ -1,13 +1,12 @@
 function [DataOut] = xASL_num2str(DataIn, f, bConcatenate, strDelimiter)
```

Comment:
I know that this was there before, but can we change it to something meaningful - like 'format' or 'stringFormat'?

Reply:
I think I copied this from `num2str`, so if `stringFormat` is correct then yes, good idea. I would then also use `stringDelimiter` instead of `strDelimiter`

Reply:
ea1ec1d